lib/ansible/cli/doc.py

```diff
@@ -89,8 +88,8 @@ def _load_argspec(self, role_name, collection_path=None, role_path=None):
         else:
             raise AnsibleError("A path is required to load argument specs for role '%s'" % role_name)
 
-        loader = DataLoader()
-        return loader.load_from_file(path, cache=False, unsafe=True)
+        with open(path, 'r') as f:
```

Comment:
trap ioerror/oserror and return {} if you want it to work the same as with dataloader